location-service/src/main/java/com/movie/recommender/location/model/dto/LocationCreateDTO.java

```diff
@@ -18,13 +19,17 @@ public class LocationCreateDTO {
     @DecimalMin(value = "-180.0", message = "Longitude must be between -180 and 180 degrees")
     @DecimalMax(value = "180.0", message = "Longitude must be between -180 and 180 degrees")
     private double longitude;
+
     private Long userId;
 
+    private City city;
```

Comment:
`city` field shouldn't be in `LocationCreateDTO` because it’s fetched from the geolocation API, not provided by the user